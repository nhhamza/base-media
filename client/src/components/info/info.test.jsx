import React from 'react';
import { shallow } from 'enzyme';

import InfoComponent from './info.component';

describe('Info Component', () => {
  let wrapper;

  const infoHelp = {
    title: 'YOU NEED HELP?',
    description: 'Description of Help',
  };

  beforeEach(() => {
    wrapper = shallow(<InfoComponent {...infoHelp} />);
  });

  it('should render Info component', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('the title should be "YOU NEED HELP?"', () => {
    const title = wrapper.find('.information__title').text();
    expect(title).toEqual(infoHelp.title);
  });

  it('the description should be "Description of Help"', () => {
    const description = wrapper.find('.information__description').text();
    expect(description).toEqual(infoHelp.description);
  });
});
