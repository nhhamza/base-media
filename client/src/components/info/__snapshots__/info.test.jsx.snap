// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Component should render Info component 1`] = `ShallowWrapper {}`;
